community/import-tool/src/docs/ops/import-tool.asciidoc

```diff
@@ -107,9 +119,9 @@ They have an id, a name and all have the `Actor` label.
 include::actors.csv[]
 ----
 
-Finally we have the roles, which are represented by relationships in the database.
-Here we refer to the ids used in the other data sources, which happens in the `START_ID` and `END_ID` fields.
-At the end of each row, we have put the relationship type to use.
+Finally we have the roles that an actor plays in a movie which will be represented by relationships in the database.
+In order to create a relationship between nodes we refer to the ids used in `actors.csv` and `movies.csv` in the `START_ID` and `END_ID` fields.
+We also need to provide a relationship type (in this case `ACTS_IN`) in the `:TYPE` field.
```

Comment:
@tinwelint I've updated the docs to show an example where you specify the same label for every node but the relationship file has a mandatory `:TYPE` field so I can't do an example for there yet